src/main/java/com/seailz/discordjar/model/channel/GuildChannel.java

```diff
@@ -111,4 +118,86 @@ default void addPermissionOverwrite(@NotNull PermissionOverwrite overwrite) {
+        DiscordResponse response = new DiscordRequest(
+                new JSONObject()
+                        .put("name", name),
+                new HashMap<>(),
+                URLS.POST.GUILDS.CHANNELS.CREATE_WEBHOOK.replace("{guild.id}", guild().id()).replace("{channel.id}", id()),
+                discordJv(),
+                URLS.POST.GUILDS.CHANNELS.CREATE_WEBHOOK,
+                RequestMethod.POST
+        ).invoke();
+        return IncomingWebhook.decompile(response.body(), discordJv());
+    }
+
+    /**
+     * Gets a Webhook a channel has in effect.
+     * @param id The id of the Webhook.
+     * @param token The token of the Webhook.
+     * @return A {@link com.seailz.discordjar.model.webhook.Webhook} object.
+     * @throws DiscordRequest.UnhandledDiscordAPIErrorException Thrown when an unexpected error is returned from the Discord API.
+     */
+    default Webhook getWebhookById(long id, String token) throws DiscordRequest.UnhandledDiscordAPIErrorException {
+        DiscordResponse response = null;
+        response = new DiscordRequest(
+                new JSONObject(),
+                new HashMap<>(),
+                URLS.GET.GUILDS.CHANNELS.GET_CHANNEL_WEBHOOK.replace("{webhook.id}", String.valueOf(id)).replace("{webhook.token}", token),
+                discordJv(),
+                URLS.GET.GUILDS.CHANNELS.GET_CHANNEL_WEBHOOK,
+                RequestMethod.GET
+        ).invoke();
+        return Webhook.decompile(response.body(), discordJv());
+
+    }
+
+    /**
+     * Gets a Webhook a channel has in effect.
+     * @param id The id of the Webhook.
+     * @param token The token of the Webhook.
+     * @return A {@link com.seailz.discordjar.model.webhook.Webhook} object.
+     * @throws DiscordRequest.UnhandledDiscordAPIErrorException Thrown when an unexpected error is returned from the Discord API.
+     */
+    default Webhook getWebhookById(String id, String token) throws DiscordRequest.UnhandledDiscordAPIErrorException {
+        DiscordResponse response = null;
+        response = new DiscordRequest(
+                new JSONObject(),
+                new HashMap<>(),
+                URLS.GET.GUILDS.CHANNELS.GET_CHANNEL_WEBHOOK.replace("{webhook.id}", id).replace("{webhook.token}", token),
+                discordJv(),
+                URLS.GET.GUILDS.CHANNELS.GET_CHANNEL_WEBHOOK,
+                RequestMethod.GET
+        ).invoke();
+        return Webhook.decompile(response.body(), discordJv());
+    }
+
+    /**
+     * Gets the Webhooks that the channel has in effect.
+     * @return A {@link java.util.List} of {@link com.seailz.discordjar.model.webhook.Webhook} objects.
+     * @throws DiscordRequest.UnhandledDiscordAPIErrorException Thrown when an unexpected error is returned from the Discord API.
+     */
+    default List<Webhook> getWebhooks() throws DiscordRequest.UnhandledDiscordAPIErrorException {
```

Comment:
Put this method in the impl class of GuildChannel
```suggestion
    List<Webhook> getWebhooks() throws DiscordRequest.UnhandledDiscordAPIErrorException;
```